File: Java/MochiBasah.java
public class MochiBasah extends ProdukMochi {
    private String isian, ukuran, warna;

    public MochiBasah(String kode, String nama, int harga, String isian, String ukuran, String warna) {
        super(kode, nama, harga);
        this.isian = isian;
        this.ukuran = ukuran;
        this.warna = warna;
    }

    public String getIsian() { return isian; }
    public String getUkuran() { return ukuran; }
    public String getWarna() { return warna; }
}

File: CPP/MochiBasah.cpp
#include "MochiBasah.h"

MochiBasah::MochiBasah(string k, string n, int h, string i, string u, string w)
    : ProdukMochi(k, n, h), isian(i), ukuran(u), warna(w) {}

string MochiBasah::getIsian() { return isian; }
string MochiBasah::getUkuran() { return ukuran; }
string MochiBasah::getWarna() { return warna; }
